sdk/Cargo.toml

```
@@ -67,6 +67,7 @@ heck = { version = "0.4.1", default-features = false, optional = true }
 instant = { version = "0.1.12", default-features = false, optional = true }
 iota-ledger-nano = { version = "1.0.0-alpha.4", default-features = false, optional = true }
 iota_stronghold = { version = "2.0.0", default-features = false, optional = true }
+jammdb = { version = "0.10.0", optional = true }
```

We add this to all crates so we don't get features we don't actually need, for consistency it should also be here even if there are not features at all (at least currently)
```suggestion
jammdb = { version = "0.10.0", default-features = false, optional = true }
```